test/runtime/samples/pre-tag/.editorconfig

```diff
@@ -0,0 +1,2 @@
+[main.svelte]
+trim_trailing_whitespace = unset
```

Comment:
?
remove the extra file?

Reply:
This is necessary to edit the blank test code before the line break in the editor.

https://github.com/sveltejs/svelte/pull/7280/files#diff-d5881bd31993ce2fe17bf9174468471046fe3c66fb8c1cf2b43a842aa41fbd77R49-R50

Without it, the test code will change unintentionally when someone edits the file in the future.

Reply:
i wonder is it there's some plugins that do some auto formatting before u save?

in any case, the unintentional changes will lead to git changes / test failed right?

Reply:
You are using EditorConfig. There is a configuration file at the root of the project.

https://github.com/sveltejs/svelte/blob/master/.editorconfig

The added file is for overwriting its configuration.

Reply:
> in any case, the unintentional changes will lead to git changes / test failed right?

If trailing whitespace is removed from the .svelte file, the test on the next line will fail.

https://github.com/sveltejs/svelte/pull/7280/files#diff-0a1ca6ee0ecfd38066746f565bce75c4f5ce64d2b5ac5505330c2e4a85769547R52